src/string.cr

```diff
@@ -1415,6 +1415,18 @@ class String
     cmp == 0 ? (bytesize <=> other.bytesize) : cmp
   end
 
+  # Test whether `str` matches `regex`.
```

Comment:
I think so far we use "\*str\* matches \*regex\*" to refer to argument names. In more places than not anyway.